test/eks/release-cluster.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+region=eu-central-1
+export KUBECONFIG=eks-kubeconfig
+
+cluster=$(cat cluster-name)
+ng=$(eksctl get nodegroup --cluster $cluster -r $region -o json | jq -r '.[0].Name')
+
+echo "scaling $cluster ng $ng to 0"
+eksctl scale nodegroup -r $region --cluster $cluster -n $ng -N 0
+
+echo "releasing cluster lock from $cluster"
+kubectl annotate deployment lock lock-
```

Comment:
It might be worth deleting the `cluster-name` file here to avoid accidentally using it when we have some bug but it still refers to a previous run. I'm really not sure how this can happen, though... so maybe it doesn't matter.

Reply:
Sure, can't hurt us.